docs/developers-guide/driver-changelog.md

```diff
@@ -70,6 +70,9 @@ title: Driver interface changelog
   removed in 0.51.0 or later. You can easily implement `metabase.driver/db-default-timezone` directly, and use
   `metabase.driver.sql-jdbc.execute/do-with-connection-with-options` to get a `java.sql.Connection` for a Database.
 
+- `metabase.driver/prettify-native-form` was added to enable driver developers use native form formatting
```

Comment:
I think we should use the word prettify or something similar so as not to confuse things with other formatting like HoneySQL to SQL string.